packages/hub/schema/index.js

```diff
@@ -199,6 +199,28 @@ class Schema {
     }
   }
 
+  async hasLoginAuthorization(user={}) {
```

Comment:
maybe `potentialSession`?

Reply:
I see what you meant, but we can just move the `canLogin` check down below the session creation, so a session will be available, even though it's newly created.

While that won't create an opportunity for caching here, it would make caching possible when/if we decide to start applying this grant even to _keep_ yourself logged in.

Reply:
sure

Reply:
oh yeah, potentialSession is a better name too.